### PR TITLE
Hello Microverse project for code review practice.

### PR DESCRIPTION
**Pull request by demix007**
## The changes implemented in this pull request include:

- Created a new branch.

- Added a .gitignore file.

- Set up a linter for HTML & CSS.

-  Added a README.md file.

- Added an index.html file that includes a header with the text "Hello Microverse!". 

- Added a style.css file.

- Added a test.md file with text "This file should be ignored by git".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly be noticed that your README file should have the same fonts size and style as the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md).

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @demix007, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- No Linter Error ✔️
- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
- [x] Kindly be noticed that Pull Request should have a descriptive title telling you what changes are inside.
- [ ] Also notice that Pull Request should have a summary describing introduced changes in more detail.
- [ ] Kindly be noticed that the `test.md` file should not be in the GitHub repo please remove it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by demix007** [comment]
Thank you @ShahierNashaat for the feedback, the necessary changes have been made. Kindly help with the review.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Abenezer-Tilahun) [inline README.md:42]: - [optional ] The MIT LICENSE link is broken try to fix it and add LICENSE file in your directory.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @demix007 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

almost

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail Just like this
(In this milestone I Implemented the f/f :- 
  Setup linter for HTML and CSS, Add a .gitignore file, Add a .html file includes a header with the text "Hello Microverse!", Add  .css file that includes one class with styles for your header , add A test.md file with the text "This file should be ignored by git".)
 )

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @demix007 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- no linter errors
- good PR description 
- good README
- correct github flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**